styles/themes/classic.css

```diff
@@ -330,7 +331,7 @@ button.selected,
 	background: -ms-linear-gradient(left, #000080 0%, #1084d0 100%); /* IE10+ */
 	background: linear-gradient(to right, #000080 0%, #1084d0 100%); /* W3C */
 	filter: progid:DXImageTransform.Microsoft.gradient( startColorstr='#000080', endColorstr='#1084d0', GradientType=1 ); /* IE6-9 */
-	
```

Comment:
I'm using atom so its getting rid of extra white space

Reply:
Yeah, don't worry about it. I should probably set up a formatter to get rid of all of the rest of it in one go (and make various other things more consistent).

Reply:
I can set up Prettier to auto format with a .prettierc